install_files/securedrop-app-code/debian/postinst

```diff
@@ -93,6 +93,14 @@ function adjust_wsgi_configuration {
             perl -pi -e 's/^WSGIProcessGroup journalist.*\n//' "$journalist_conf"
         fi
     fi
+    # Create the WSGI so file link if only on Python3.8
+    wsgi_so_link="/opt/venvs/securedrop-app-code/lib/python3.8/site-packages/mod_wsgi/server/mod_wsgi-py38.so"
+    wsgi_so="/opt/venvs/securedrop-app-code/lib/python3.8/site-packages/mod_wsgi/server/mod_wsgi-py38.cpython-38-x86_64-linux-gnu.so"
+    if test -f "/usr/bin/python3.8"; then
+        if test ! -f  $wsgi_so_link; then
```

Comment:
The `test -f` conditional on L100 will always fail, since the target is a symlink, not a file. Rather than `-f`, `-L` would be a better test.